python/ccf/clients.py

```diff
@@ -543,6 +550,8 @@ class CCFClient:
     :param int connection_timeout: Maximum time to wait for successful connection establishment before giving up.
     :param str description: Message to print on each request emitted with this client.
     :param bool ws: Use WebSocket client (experimental).
+    :param bool disable_client_auth: Do not use the provided client identity to authenticate. The client identity will still be used to sign if ``signed`` is set to True when making requests.
```

Comment:
As discussed, this can probably go as only the key needs to be specified to be able to sign the HTTP request.

Reply:
Probably not in fact, because we want the cert so we can digest it and calculate the key id.

Reply:
The public key could also be derived from the private key directly, so passing the certificate is still optional.

Reply:
Well no, we want to digest the certificate specifically, and that can’t be derived from the key pair.